src/translations/translationMetadata.json

```diff
@@ -2,6 +2,9 @@
   "af": {
     "nativeName": "Afrikaans"
   },
+  "am": {
```

Comment:
The language code is incorrect. It should be hy.

![image](https://user-images.githubusercontent.com/1444314/64367669-7b933100-cfcd-11e9-91b3-51d23c72f976.png)

Reply:
Fixed, thanks!